tests/prague/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -470,3 +471,214 @@ def test_eof_calls_legacy_mstore(
         post=post,
         tx=tx,
     )
+
+
+@pytest.mark.parametrize(
+    ["opcode", "suffix"],
+    [
+        [Op.EXTCALL, [0, 0, 0]],
+        [Op.EXTDELEGATECALL, [0, 0]],
+        [Op.EXTSTATICCALL, [0, 0]],
+    ],
+    ids=["extcall", "extdelegatecall", "extstaticall"],
+)
+@pytest.mark.parametrize(
+    "destination_opcode, destination_suffix",
+    [[Op.REVERT, [0, 0]], [Op.INVALID, []]],
+    ids=["revert", "invalid"],
+)
+@pytest.mark.parametrize("destination_is_eof", [True, False])
+def test_eof_calls_revert_abort(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    sender: EOA,
+    opcode: Op,
+    suffix: list[int],
+    destination_opcode: Op,
+    destination_suffix: list[int],
+    destination_is_eof: bool,
+):
+    """Test EOF contracts calling contracts that revert or abort"""
+    env = Environment()
+
+    destination_contract_address = pre.deploy_contract(
+        Container.Code(destination_opcode(*destination_suffix))
```

Comment:
```suggestion
        Container.Code(destination_opcode(offset=0, size=0))
```
I think this should work too, it will default to not passing any arguments to `Op.INVALID`.

The keyword argument logic is: if there is at least one keyword argument, consume the ones provided, ignore the unknown ones, and use the default (normally 0, or `Op.GAS` for gas argument of legacy calls) for the ones missing.